### PR TITLE
add info about suggesting tutorials

### PR DESCRIPTION
**Pull request by fharper**

### CODE REVIEW

File: learn/getting_started/whats_next.md [hunk 1]
```diff
@@ -18,4 +18,4 @@ And additional resources here:
 - [How-To Guides](/create/how_to)
 - [Postman Collection](/create/how_to/postman_collection.md): it can be tedious to re-write every route when wanting to try out an API. Try out MeiliSearch with our collection in Postman
 
-More tutorials are currently in development and will be available soon.
+More tutorials are currently in development and will be available soon. If you want to suggest a tutorial, please [create an issue](https://github.com/meilisearch/devrel/issues/new) in the [Developer Relations repository](https://github.com/meilisearch/devrel/).
```
Comment (react-learner) [inline learn/getting_started/whats_next.md:21]: Wouldn't it make more sense to direct them to the documentation repo?

**Comment by fharper** [comment]
Depends, I thought that we wanted to have the tutorials in the blog no? I mean, the non crucial ones.

**Comment by react-learner** [comment]
@fharper Agreed, but I think it's confusing to call for them in the documentation if they won't go there, at least not without further explanation. I think we shouldn't add this here, it would make more sense in the new contribution section.

**Comment by fharper** [comment]
Make sense

**Pull request closed without merging**